webapp/src/components/Staking/LiquidityGaugeModal/ClaimActionModal.tsx

```diff
@@ -130,8 +158,70 @@ const ClaimActionModal: React.FC<ClaimActionModalProps> = ({
     return `${startTime.days()}D ${startTime.hours()}H ${startTime.minutes()}M`;
   }, [lg5Data, lg5DataLoading, loadingText]);
 
+  const rewards: {
+    totalPoolRewards: string;
+    baseRewards: string;
+    boostedRewardsMultiplier: string;
+    boostedRewardsAmount: string;
+  } = useMemo(() => {
+    if (apysLoading) {
+      return {
+        totalPoolRewards: loadingText,
+        baseRewards: loadingText,
+        boostedRewardsMultiplier: loadingText,
+        boostedRewardsAmount: loadingText,
+      };
+    }
+
+    const totalPoolRewards = lg5Data
+      ? parseFloat(formatBigNumber(lg5Data.poolRewardForDuration))
+      : 0;
```

Comment:
Can we show a `-` for loading?  When a user see their balance/rewards is `0` because the connection failed, they will freak out